.circleci/config.yml

```diff
@@ -186,6 +191,27 @@ jobs:
           name: Publish on npm
           command: npm publish
 
+  integration_tests:
+    executor: integration-executor
+    working_directory: *sdk_working_dir
+    steps:
+      - attach_workspace: *attach_options
+      - run:
+          name: Restore dependencies
+          command: npm ci
+      - run:
+          name: Run integration tests
+          command: source /etc/profile.d/smartcontracts.sh && npm run test:integration -- --ci --runInBand --reporters=default --reporters=jest-junit
```

Comment:
sourcing the script here is required because otherwise the `TTT_TOKEN_ADDRESS` will be undefined and the tests will fail